types/k6/experimental/fs.d.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ * The k6 File System (fs) module provides an API for handling file operations 
```

Comment:
I know that is obvious, but how about explicitly naming the module is experimental? 
```suggestion
 * The k6 File System (fs) experimental module provides an API for handling file operations 
```